openchemistry/_calculation.py

```diff
@@ -72,10 +72,23 @@ def __init__(self, _id=None, properties=None, molecule_id=None):
         self._id = _id
         self._properties = properties
         self._molecule_id = molecule_id
+        self._optimized_geometry_id = None
 
     def data(self):
         return self._provider.cjson
 
+    @property
+    def optimized_geometry_id(self):
+        if not self._optimized_geometry_id:
+            # Try to get it...
+            result = GirderClient().get('calculations/%s' % self._id)
+            if 'optimizedGeometryId' in result:
+                self._optimized_geometry_id = result['optimizedGeometryId']
+            else:
+                print('None')
```

Comment:
Shouldn't we raise an exception here or return `None`

Reply:
Yeah, so `None` gets returned below since `self._optimized_geometry_id` will be `None`. But I got it to print `None` as well, so that if the user does `result.optimized_geometry_id` in a notebook, and it returns `None`, that they will see `None` (rather than nothing happening)

Reply:
Wouldn't setting _optimized_geometry_id to None be appropriate if there is no ID for us? I would rather check for None than catch exceptions.